### PR TITLE
[onednn] Update to 2.3.2

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates onednn to 2.3.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  As before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run ./vcpkg x-add-version --all and committed the result?  
  Yes

### CODE REVIEW

File: ports/onednn/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
 {
   "name": "onednn",
-  "version-string": "2.0",
-  "port-version": 1,
+  "version-semver": "2.3.2",
```
Comment (chausner) [inline ports/onednn/vcpkg.json:3]: Adhering to semver according to https://github.com/oneapi-src/oneDNN/issues/734#issuecomment-635100472.

File: ports/onednn/portfile.cmake [hunk 2]
```diff
@@ -3,27 +3,28 @@ vcpkg_fail_port_install(ON_ARCH "x86" "arm" ON_TARGET "uwp")
...
+vcpkg_cmake_config_fixup(PACKAGE_NAME dnnl CONFIG_PATH lib/cmake/dnnl)
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 # Copyright and license
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+
+
```
Comment (NancyLi1013) [inline ports/onednn/portfile.cmake:30]:
```suggestion
```
Please remove these two extra space lines.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: versions/o-/onednn.json [hunk 3]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "deafa9f4ff5629d7d5fc71964c470c6bec3db65f",
```
Comment (NancyLi1013) [inline versions/o-/onednn.json:4]:
```suggestion
      "git-tree": "3d3bacca42ae3bc1a41217c50f23953f13220732",
```

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Too bad we can't teach onednn to update itself :D. Thanks!

**Pull request merged**